### ISSUE TITLE
yo trails:whatevergenerator fails

### ISSUE THREAD

**Issue by fernandolguevara** [trailsjs/trails#65]
i'm executing this command **yo trails:model Customer** and i get this error, same error for the others generators

```                                                                                                                             
module.js:328
    throw err;
    ^

Error: Cannot find module 'yeoman-generator/node_modules/path-exists'
    at Function.Module._resolveFilename (module.js:326:15)
    at Function.Module._load (module.js:277:25)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/f00lg/.nvm/versions/node/v5.3.0/lib/node_modules/generator-trails/generators/app/util.js:13:46)
    at Module._compile (module.js:398:26)
    at Object.Module._extensions..js (module.js:405:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
```

**Comment by jaumard** [comment]
Thanks for the report, this issue is related to https://github.com/trailsjs/generator-trails/issues/18 and be fix soon.

**Comment by YMA-MDL** [comment]
is there any way to use an older version to try something working? 
Should this issue also impact the way versions are released ?

**Comment by jaumard** [comment]
Fix on `generator-trails` v0.0.10
